dockerfiles/ubi9.4/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN --mount=type=secret,id=redhat_pw,uid=0 \
     --username $(cat /run/secrets/redhat_user) \
     --password $(cat /run/secrets/redhat_pw)  && \
   GPU_ENABLED=false /deps/base.sh  && \
-  # /deps/tesseract.sh && \
+  /deps/tesseract.sh && \
```

Comment:
This was accidentally commented out in #27 , but doesn't affect CI at all since that image is only built locally.